lib/IDE/Refactoring.cpp

```diff
@@ -2601,6 +2601,77 @@ bool RefactoringActionLocalizeString::performChange() {
   EditConsumer.insertAfter(SM, Target->getEndLoc(), ", comment: \"\")");
   return false;
 }
+    
+bool RefactoringActionDocCommentBoilerplate::isApplicable(ResolvedCursorInfo Tok, DiagnosticEngine &Diag) {
+#warning Implement here
```

Comment:
Delete these #warnings

Reply:
Do you want the implementation for constructors and subscripts in a separate PR? I just have those warnings so that I can quickly find the file in Xcode.

Reply:
Ah! Sorry I thought they were left over. You can add subscripts and initializers in this PR — just ping me when you think it’s ready 😁

Reply:
Now that I understand the process, implementing initializers and subscripts should be a lot faster! Thanks for all your feedback and help!